lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -1502,6 +1503,23 @@ def _calc_coord(self, xv, yv, renderer=None):
         p2 = p1 - scale*vec
         return p2, pane_idx
 
+    def _arcball(self, p):
+        """
+        Convert a point p = [x, y] to a point on a virtual trackball
```

Comment:
Minor formatting:
```suggestion
        Convert a point p = (x, y) to a point on a virtual trackball.
        
```

Reply:
Yes; and I think we can leave out the "p = " too, if we change to `x, y` as arguments.